README.md

```diff
@@ -1,39 +1,27 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+![](https://img.shields.io/badge/Microverse-blueviolet) ![](https://img.shields.io/badge/-HTML-orange) ![](https://img.shields.io/badge/-CSS-blue) ![](https://img.shields.io/badge/-JavaScript-yellow) ![](https://img.shields.io/badge/-Luxon%202.x-CA82FF)
 
-# Project
-> About this project.
+# Awesome Books with ES6 and modules
+> A basic app project built with HTML, CSS and JS ES6 to keep track of awesome books.
 
 
 ## Built With
 
-- **HTML & CSS best practices:** Correct use of tags, elements, properties and syntax.
+- **HTML/CSS and JS best practices:** Correct use of tags, elements, properties and syntax.
 - **GitHub flow:**  Correct use of Branches for deployment and features development.
 - **Linters Check:** Local and Pull Request check for errors, bugs and stylistic errors in code.
+- **Luxon 2.x** for JavaScript dates and times.
 
 ## Live Demo 
 
-[Live Demo Link](https://raminka13.github.io/M1-Portfolio/)
+[Awesome Books Demo](https://karlavdelgadof.github.io/Awesome-Books-w-modules/)
 
 ## Authors
 
-👤 **Raul A Ospina**
-
-- GitHub: [@raminka13](https://github.com/raminka13)
-- Twitter: [@raminka13](https://twitter.com/raminka13)
-- LinkedIn: [Raul Ospina](http://linkedin.com/in/raul-ospina-83232614)
-
-
-## Collaborators
-
-- [Name](URL)
-- [Name](URL)
-
-
-## Project Milestones
-
-
-- Milestone 1
+👤 **Karla Delgado**
 
+- GitHub: [@karlavdelgadof](https://github.com/karlavdelgadof)
+- Twitter: [@karlavdelgadof](https://twitter.com/karlavdelgadof)
+- LinkedIn: [Karla Delgado](https://www.linkedin.com/in/karla-delgado-613a32239/)
 
 ## Show your support
 
```

Comment:
- [ ] Kindly add more sections in your README as we have on the [Microverse template](https://github.com/microverseinc/readme-template), like the Getting Started section describing how to set up and run the app locally.

Reply:
I'm sorry but this is not a project requirement since app doesn't require installation/cloning guide